docs/group_faqs/gf_faqs.rst

```diff
@@ -25,27 +23,27 @@ On the PubSub+ Cloud `Create Service <https://console.solace.cloud/services/crea
 .. image:: ../img/faqs_enterprise_access_1.png
    :target: https://console.solace.cloud/services/create
 
-How are plans used?
+How are service types used?
 ~~~~~~~~~~~~~~~~~~~
 
-There are two ways you could use an enterprise plan, you can ask for a Prepaid Subscription or for On-Demand services.
+There are two ways you could use an enterprise service type, you can ask for a Prepaid Subscription or for On-Demand services.
 
-How do I know what plan is best for my needs?
+How do I know what service type is best for my needs?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To understand the details of the plans offered by PubSub+ cloud, please visit the pricing page on our site, the plans differ on number of connections
+To understand the details of the service types offered by PubSub+ cloud, please visit the pricing page on our site, the service types differ on number of connections
 Throughput, Usage and Storage:
```

Comment:
Put something after connections saying "which are customizable for Enterprise service types."